RyEngine/src/Core/Input/Mouse.cpp

```diff
@@ -15,9 +14,23 @@ namespace RE
 	{
 		ASSERT(eventType == EventType::RE_MOUSE_DOWN || eventType == EventType::RE_MOUSE_UP || eventType == EventType::RE_MOUSE_MOVE || eventType == EventType::RE_MOUSE_SCROLL);
 
+		if (_mEventCallbacks.find(eventType) != _mEventCallbacks.end())
```

Comment:
Should this check happen before the ASSERT?

Reply:
I'm realizing now this TODO is not correct. The reason _mEventCallbacks is a map of lists is because I want to be able to add any number of callbacks for the same event. There could be multiple things listening for a particular mouse event.